csharp/src/Apache.Arrow/Arrays/ListArray.cs

```diff
@@ -69,25 +83,28 @@ public ListArray Build(MemoryAllocator allocator = default)
 
                 return new ListArray(DataType, Length - 1,
                     ValueOffsetsBufferBuilder.Build(allocator), ValueBuilder.Build(allocator),
-                    new ArrowBuffer(), 0, 0);
+                    ValidityBufferBuilder.Build(allocator).ValueBuffer, NullCount, 0);
```

Comment:
Similar comment on the Validity buffer being `Empty` when `NullCount == 0`.

Reply:
Fixed